apps/fluent-tester/src/TestComponents/Notification/NotificationTest.tsx

```diff
@@ -9,156 +14,262 @@ const svgProps: SvgIconProps = {
+    <View>
+      <StyledPicker
+        prompt="Variant"
+        selected={variant}
+        onChange={setVariant}
+        collection={NotificationVariants.filter((variant) => !variant.endsWith('Bar'))}
+      />
+      <TextInput value={title} onChangeText={setTitle} style={styles.textInput} />
+      <TextInput value={message} onChangeText={setMessage} style={styles.textInput} />
+      <TextInput value={action} onChangeText={setAction} style={styles.textInput} />
+      <View style={commonStyles.switch}>
+        <Text>Show icon </Text>
+        <Switch value={showIcon} onValueChange={setShowIcon} />
+      </View>
+      <View style={commonStyles.switch}>
+        <Text>Show title </Text>
+        <Switch value={showTitle} onValueChange={setShowTitle} />
+      </View>
+      <View style={styles.notification}>
+        <Notification
+          variant={variant}
+          icon={showIcon ? iconProps : undefined}
+          title={showTitle ? title : undefined}
+          action={action}
+          onPress={() => {
+            console.log('Notification tapped');
+          }}
+          onActionPress={() => {
+            console.log('Notification action tapped');
+          }}
+        >
+          {message}
+        </Notification>
+      </View>
+    </View>
   );
 };
 
-const PrimaryBarTest: React.FunctionComponent = () => {
+const CustomBar: React.FunctionComponent = () => {
+  const [variant, setVariant] = React.useState<NotificationVariant>('primaryBar');
+  const [message, setMessage] = React.useState('Updating...');
+
   return (
-    <Notification
-      variant={'primaryBar'}
-      onPress={() => {
-        console.log('Notification tapped');
-      }}
-    >
-      Updating...
-    </Notification>
+    <View>
+      <StyledPicker
+        prompt="Variant"
+        selected={variant}
+        onChange={setVariant}
+        collection={NotificationVariants.filter((variant) => variant.endsWith('Bar'))}
+      />
+      <TextInput value={message} onChangeText={setMessage} style={styles.textInput} />
+      <View style={styles.notification}>
+        <Notification
+          variant={variant}
+          onPress={() => {
+            console.log('Notification tapped');
+          }}
+        >
+          {message}
+        </Notification>
+      </View>
+    </View>
   );
 };
 
-const PrimaryOutlineBarTest: React.FunctionComponent = () => {
+const PrimaryWithAutoHide: React.FunctionComponent = () => {
+  const [visible, setVisible] = React.useState(false);
+  const onButtonPress = () => setVisible(!visible);
+
+  const [hidden, setHidden] = React.useState<boolean>(!visible);
+  const height = React.useRef(new Animated.Value(0)).current;
+
+  const show = () => {
+    setHidden(false);
+    Animated.sequence([
+      Animated.timing(height, {
+        toValue: SHOW_HEIGHT,
+        duration: NOTIFICATION_SHOW_DURATION,
+        useNativeDriver: USE_NATIVE_DRIVER_IOS,
+        easing: Easing.elastic(NOTIFICATION_BOUNCINESS),
```

Comment:
what are some of the option here? not sure it is the pr captioning issue but I would expect there should be more damping effect after notification is shown?

Reply:
So `toValue` is the height that the notification will reach after `duration` milliseconds. There is no option for damping effect in Animations but it can be mocked with an elastic easing, which causes the notification to reach the `toValue` height, overshoot, and then return to the `toValue` height.

Reply:
https://reactnative.dev/docs/easing Does this help?

Reply:
Could we use spring anywhere? I see that has a configurable damping
https://reactnative.dev/docs/animated#spring

Reply:
hm spring has a configurable damping but no duration